DevEdu.API/DevEdu.Business.Tests/RatingServiceTests.cs

```diff
@@ -178,42 +235,76 @@ public void AddStudentRating_StudentDoesntBelongToGroup_ValidationException()
+        public void DeleteStudentRating_AuthorUserIsTeacher_StudentRatingDeleted()
         {
             //Given
             var expectedStudentRatingDto = RatingData.GetOutputStudentRatingDto();
             var studentRatingId = expectedStudentRatingDto.Id;
-            var authorUserId = 1;
+            var authorUserInfo = new UserIdentityInfo
+            {
+                UserId = 1,
+                Roles = new List<Role> { Role.Teacher }
+            };
             var groupId = expectedStudentRatingDto.Group.Id;
             var usersInGroup = UserData.GetListUsersDto();
 
             _ratingRepoMock.Setup(x => x.SelectStudentRatingById(studentRatingId)).Returns(expectedStudentRatingDto);
             _userRepoMock.Setup(x => x.GetUsersByGroupIdAndRole(groupId, (int)Role.Teacher)).Returns(usersInGroup);
 
             //When
-            _sut.DeleteStudentRating(studentRatingId, authorUserId);
+            _sut.DeleteStudentRating(studentRatingId, authorUserInfo);
 
             //Than
             _ratingRepoMock.Verify(x => x.SelectStudentRatingById(studentRatingId), Times.Once);
             _ratingRepoMock.Verify(x => x.DeleteStudentRating(studentRatingId), Times.Once);
             _userRepoMock.Verify(x => x.GetUsersByGroupIdAndRole(groupId, (int)Role.Teacher), Times.Once);
         }
 
+        [Test]
+        public void DeleteStudentRating_AuthorUserIsAdmin_StudentRatingDeleted()
+        {
+            //Given
+            var expectedStudentRatingDto = RatingData.GetOutputStudentRatingDto();
+            var studentRatingId = expectedStudentRatingDto.Id;
+            var authorUserInfo = new UserIdentityInfo
+            {
+                UserId = 4,
+                Roles = new List<Role> { Role.Admin }
+            };
+            var groupId = expectedStudentRatingDto.Group.Id;
+            var usersInGroup = UserData.GetListUsersDto();
+
+            _ratingRepoMock.Setup(x => x.SelectStudentRatingById(studentRatingId)).Returns(expectedStudentRatingDto);
+            _userRepoMock.Setup(x => x.GetUsersByGroupIdAndRole(groupId, (int)Role.Teacher)).Returns(usersInGroup);
+
+            //When
+            _sut.DeleteStudentRating(studentRatingId, authorUserInfo);
+
+            //Than
+            _ratingRepoMock.Verify(x => x.SelectStudentRatingById(studentRatingId), Times.Once);
+            _ratingRepoMock.Verify(x => x.DeleteStudentRating(studentRatingId), Times.Once);
+            _userRepoMock.Verify(x => x.GetUsersByGroupIdAndRole(It.IsAny<int>(), It.IsAny<int>()), Times.Never);
+        }
+
         [Test]
         public void DeleteStudentRating_TeacherDoesntAuthorizeToGroup_AuthorizationException()
         {
             //Given
             var expectedStudentRatingDto = RatingData.GetOutputStudentRatingDto();
             var studentRatingId = expectedStudentRatingDto.Id;
-            var authorUserId = 4;
+            var authorUserInfo = new UserIdentityInfo

```

Comment:
please make a method that will return such objects
userId and userRole will be the parameters of this method